cairo/src/device.rs

```diff
@@ -157,7 +157,7 @@ impl Device {
             let status = ffi::cairo_device_acquire(self.to_raw_none());
             status_to_result(status)?;
         }
-        Ok(DeviceAcquireGuard { 0: self })
```

Comment:
I didn't consider that this was valid syntax :rofl:

Reply:
It's very convenient when writing macros. :3